src/components/SearchBar/components/AddressBook/AddressBook.tsx

```diff
@@ -1,26 +1,13 @@
 import React, { memo, useCallback, useMemo, useState } from "react";
-import {
-  Button,
-  DialogContent,
-  FormGroup,
-  IconButton,
-  Input,
-  Paper,
-  Stack,
-  styled,
-  Table,
-  TableBody,
-  TableCell,
-  TableContainer,
-  TableHead,
-  TableRow,
-} from "@mui/material";
+import { Button, DialogContent, FormGroup, IconButton, Input, Stack, styled } from "@mui/material";
 import ImportContactsIcon from "@mui/icons-material/ImportContacts";
 import JUPDialog from "components/JUPDialog";
 import { useSnackbar } from "notistack";
 import { messageText } from "utils/common/messages";
 import { isValidAddress } from "utils/validation";
 import useBreakpoint from "hooks/useBreakpoint";
+import JUPTable from "components/JUPTable";
+import { IHeadCellProps, ITableRow } from "components/JUPTable/JUPTable";
```

Comment:
consider moving these interfaces into the components/JUPTable location. aka, move them into the index.ts file.

also implies that they would be required to be imported within the component itself, which is easy to do.

Reply:
fixed